book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Please add a method for adding rentals. It will ensure that there is a relationship between the Book using the intermediate class Rental.

See example:
```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
end
```

Kindly note that the `self` keyword is referencing the `Book` class.